analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -302,6 +292,27 @@ def add_arguments_to_parser(parser):
                                     "'CodeChecker analyzers --dump-config "
                                     "clang-tidy' command.")
 
+    analyzer_opts.add_argument('--analyzer-config',
+                               dest='analyzer_config',
+                               nargs='*',
+                               default=argparse.SUPPRESS,
```

Comment:
We should go with the default for clang-tidy to show findings in headers (but not from system headers).
so the default should be:
--analyze-config clang-tidy:HeaderFilterRegex=*

and we should add an explicit option to take all configuration from .tidy-config files:
clang-tidy:take-config-from-directory


We should describe all this config behaviour  in the analyzer user guide.

Reply:
I think this should be done in #2690, because this is another issue.